OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -41,24 +43,33 @@
 public class CourseUnitNavigationActivity extends CourseBaseActivity implements CourseUnitVideoFragment.HasComponent {
 
     protected Logger logger = new Logger(getClass().getSimpleName());
-
+    @Inject
+    LastAccessManager lastAccessManager;
     private DisableableViewPager pager;
     private CourseComponent selectedUnit;
-
     private List<CourseComponent> unitList = new ArrayList<>();
     private CourseUnitPagerAdapter pagerAdapter;
-
     @InjectView(R.id.goto_next)
     private Button mNextBtn;
     @InjectView(R.id.goto_prev)
     private Button mPreviousBtn;
+    @InjectView(R.id.goto_next_container)
+    private LinearLayout mNextBtnContainer;
```

Comment:
`mBtnNextUnit` is a better variable name.
We call 'Unit' to a single presentable entity, keep that in naming conventions.